kadi/scripts/validate_states.py

```diff
@@ -47,6 +51,13 @@ def get_opt():
         action="store_true",
         help="Include in-work events in validation (for checking new events)",
     )
+    parser.add_argument(
+        "--email",
+        action="append",
+        dest="emails",
+        default=[],
+        help='Email address for notification (multiple allowed, use "TEST" for testing)',
```

Comment:
Oh, I don't know if it is useful to document in here that this TEST thing is a feature of acdc.common.send_mail where it just logs the output.

Reply:
It will be documented when that function gets a new home in ska_helpers.